addons/web/static/tests/core/datetime/datetime_input_tests.js

```diff
@@ -484,6 +484,19 @@ QUnit.module("Components", ({ beforeEach }) => {
         assert.strictEqual(input.value, "08/02/1997 15:45:05");
     });
 
+    QUnit.test("Cancel button click close datetime picker", async function(assert) {
+        assert.expect(1);
```

Comment:
not necessary in this case, there's no way this test could run and pass without doing one assertion.